<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateWidget</class>
 <widget class="QWidget" name="GenerateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>479</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Папка для выборки: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="setPath">
       <property name="text">
        <string>../set2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="browseSetPath">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="parameters">
     <property name="title">
      <string>Параметры генерации</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Количество значений в ряду:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>9001</number>
          </property>
          <property name="value">
           <number>1500</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Сэмплируемая функция:&lt;/p&gt;&lt;p&gt;a sin(2πα x) + b x + c exp(2/e γx) + d ln (δ e(x+1)) + ε,&lt;/p&gt;&lt;p&gt;где x ∈ [0; 1],&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="kWidgets" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Случайная ошибка ε ~ N (</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="errMean">
          <property name="toolTip">
           <string>Мат. ожидание</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>, </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="errDisperse">
          <property name="toolTip">
           <string>Дисперсия</string>
          </property>
          <property name="singleStep">
           <double>0.200000000000000</double>
          </property>
          <property name="value">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="countSetSize">
       <property name="text">
        <string>Рассчитать объём выборки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generate">
       <property name="text">
        <string>Генерировать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelReady">
       <property name="text">
        <string>Готово. </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSetSize">
       <property name="text">
        <string>Объём выборки:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCount">
       <property name="text">
        <string>NNN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSeries">
       <property name="text">
        <string>рядов</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>setPath</tabstop>
  <tabstop>browseSetPath</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>errMean</tabstop>
  <tabstop>errDisperse</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>setPath</sender>
   <signal>returnPressed()</signal>
   <receiver>generate</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
